acceptanceTests/testsUtils.go

```diff
@@ -103,6 +111,90 @@ func getServiceExternalIp(ctx context.Context, namespace string, service string)
 	return externalIp, nil
 }
 
+func switchKubeContextForTest(t *testing.T, newContextName string) error {
+	prevKubeContextName, err := getKubeCurrentContextName()
+	if err != nil {
+		return err
+	}
+
+	if err := setKubeCurrentContext(newContextName); err != nil {
+		return err
+	}
+
+	t.Cleanup(func() {
+		err := setKubeCurrentContext(prevKubeContextName)
+		if err != nil {
+			t.Errorf("failed to set Kubernetes context to %s, err: %v", prevKubeContextName, err)
+			t.Errorf("cleanup failed, subsequent tests may be affected")
```

Comment:
maybe add this log to the one above

Reply:
I prefer it like this. I think of it as a separate message.